app/models/concerns/wakeable/has_many.rb

```diff
@@ -8,6 +8,11 @@ module HasMany
       attr_accessor :has_many_label, :has_many_path
     end
 
+    def raw_aggregate_pageview_count
+      wakes_resources.reload
```

Comment:
@joeosburn when this is needed, the association has already been loaded but for whatever reason, the updating of the individual `wakes_resource` that has the new `pageview_count` is not triggering an association reload, so we have to do it manually. Quite possibly an AR bug, but I didn't care enough to investigate it further.